api/swagger.yml

```diff
@@ -2562,6 +2565,45 @@ paths:
         410:
           description: object gone (but partial metadata may be available)
 
+  /repositories/{repository}/refs/{ref}/objects/metadata:
```

Comment:
So we have:
1. `.../objects/stat` - that returns lakeFS metadata without the user metadata.
2. `.../objects/underlying_properties` - that returns the metadata on the physical object.
3. `.../objects/metadata` - that returns custom user defined metadata.

@talSofer @arielshaqed 
Do we really need all 3? Isn't it easier to consolidate some of these?

Reply:
@itaiad200 good point, this endpoint is unused `.../objects/underlying_properties.` 

How about only keeping `.../objects/stat` endpoint and add parameters to specify if the response should contain **underlying properties** and **user metadata**? the default value for these params should be false, and they will be optional. 

@itaiad200 @arielshaqed @adunuthulan WDYT?

Reply:
I've removed the third endpoint and made the default `/repositories/{repository}/refs/{ref}/objects/stat` endpoint and `/repositories/{repository}/refs/{ref}/objects/ls` endpoint return the metadata. Each has a query param that allows the user to not retrieve the user defined metadata